app/src/main/res/values-nl/strings.xml

```diff
@@ -1,66 +1,66 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">Eenvoudige Galerij</string>
+    <string name="app_name">Eenvoudige galerij</string>
```

Comment:
No

Reply:
Yes. This is the style as used in the Netherlands.

Reply:
(App) titles have every-word caps

Reply:
Not per se. Even Google has translated the News & Weather app to "Nieuws en weer".